rosidl_typeadapter_protobuf_test/CMakeLists.txt

```diff
@@ -0,0 +1,148 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(rosidl_typeadapter_protobuf_test)
+
+# Default to C++17
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
The proper way to do this is
`target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)`